IBF-Typhoon-model/src/typhoonmodel/utility_fun/Rainfall_data.py

```diff
@@ -28,13 +32,17 @@ def url_is_alive(url):
         return False
 
 
-def download_rainfall_nomads(Input_folder, path, Alternative_data_point):
+def download_rainfall_nomads(Input_folder, path, Alternative_data_point,no_data_value=29999):
     """
     download rainfall 
     """
     rainfall_path = os.path.join(Input_folder, 'rainfall/')
     if not os.path.exists(rainfall_path):
         os.makedirs(rainfall_path)
+    list_df=[]  #to store final rainfall dataframes 
+    path_admin =os.path.join(path, 'data-raw/phl_admin3_simpl2.geojson')
```

Comment:
It would be good to make the filepath string a constant at the top of the file (after all the imports), something like
`ADMIN_PATH = 'data-raw/phl_admin3_simpl2.geojson'`